index.html

```diff
@@ -46,16 +46,34 @@ <h2>Berlins Corona-Ampel</h2>
                 <div class="chart-container">
                     <canvas id="chart-reproduction" class="chart"></canvas>
                 </div>
+                <p class="chart-desc">
+                    <b>Ampelfarben beim R-Wert:</b><br/>
+                    R-Wert < 1,1 = Grün <br/>
+                    R-Wert mindestens 3 Mal <i>in Folge</i> ≥ 1,1 = Gelb <br/>
```

Comment:
You could use the more semantic `em` instead of `i`. See [here ](https://stackoverflow.com/questions/21334380/what-is-the-benefit-of-em-vs-i)for a discussion on this.